src/iff/iff_prepare.f90

```diff
@@ -193,7 +193,7 @@ subroutine precomp(env, iff_data, mol, etot, mol_num)
       !> the SP
       call singlepoint &
          &       (env, mol, chk, calc, egap, set%etemp, set%maxscciter, 2,&
-         &        exist, lgrad, acc, etot, g, sigma, res)
+         &        .false., lgrad, acc, etot, g, sigma, res)
```

Comment:
Can you shortly explain why this is now fixed?

Reply:
The boolean `exist` was never assigned a value. However, the `singlepoint` subroutine expects the boolean with `intent(in)`. Even though this was not yet causing a program crash, it might have led to memory problems in the future.